src/microbe_stage/Microbe.cs

```diff
@@ -793,14 +793,12 @@ public void Kill()
                     sceneToUse.SceneModelPath = organelle.Definition.DisplaySceneModelPath;
                     break;
                 }
-            }
 
-            // If no organelles have a scene, use mitochondrion as fallback
-            if (sceneToUse.LoadedScene == null)
-            {
-                sceneToUse.LoadedScene = SimulationParameters.Instance.GetOrganelleType(
-                    "mitochondrion").LoadedScene;
-                sceneToUse.SceneModelPath = null;
+                if (!string.IsNullOrEmpty(organelle.Definition.CorpseChunkScene))
```

Comment:
I think this if should be before check on line 788 so that even organelles that do have a model could use a cell chunk override if they want.